ArduPlane/Parameters.h

```diff
@@ -403,8 +403,10 @@ class Parameters {
     AP_Int8 waypoint_mode;
     AP_Int16 waypoint_radius;
     AP_Int16 waypoint_max_radius;
-    AP_Int16 loiter_radius;
     AP_Int16 rtl_radius;
+    
+    // Thermalling
+    AP_Int16 loiter_radius;
```

Comment:
this already exists for several flight modes, why add the Thermalling label? Why touch it at all?